plausible-yeti/client-recon
# Builds out server

#### What's this PR do?
- Create basic express server
- Sets up static pages
- Sets up routing on spec'd get/post/put requests
#### What are the important parts of the code?
- '/server/server.js' ==> basic express server
- '/server/routes/index.js' ==> router
- '/server/controllers/index.js' ==>  functions that handle routes
#### How should this be tested by the reviewer?
- Please run 'npm install' and 'npm start' in the console to start up the server
- Add console.logs as necessary to check to make sure routes are working
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

closes #8 
#### Screenshots (if appropriate)
